pylint/checkers/refactoring.py

```diff
@@ -470,8 +477,36 @@ def visit_boolop(self, node):
                              node=node,
                              args=(duplicated_name, ', '.join(names)))
 
-    @utils.check_messages('simplify-boolean-expression', 'consider-using-ternary')
+    @staticmethod
+    def _is_simple_assignment(node):
+        return (isinstance(node, astroid.Assign)
+                and len(node.targets) == 1
+                and isinstance(node.targets[0], astroid.node_classes.AssignName)
+                and isinstance(node.value, astroid.node_classes.Name))
+
+    def _check_swap_variables(self, node):
+        try:
+            assignments = [
+                node, node.next_sibling(), node.next_sibling().next_sibling()
+            ]
+        except AttributeError:
```

Comment:
Is there any particular reason not to do it this way? I found it quite nice to apply EAFP here.

Reply:
Yeah, it makes reasoning about the code a bit harder. Why is it `AttributeError`, because the node does not have a `next_sibling`? Or because the `next_sibling` is `None` and thus doesn't have a `next_sibling`? I would prefer to be explicit about what we are handling.

Reply:
The two reasonings you came up with are combined exactly what I intended to do. I personally vote for and prefer EAFP, because it is mostly a lot shorter and easier on the eye. But I am also okay with LBYL => Change incoming